.rubocop.yml

```diff
@@ -52,6 +52,10 @@ Layout/EmptyLinesAroundAttributeAccessor:
 Layout/SpaceAroundMethodCallOperator:
   Enabled: true
 
+Layout/MultilineMethodCallIndentation:
```

Comment:
Made this change because Rubocop copy was popping off about the project submissions query in the submissions controller.

I personally think [the indented](https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilinemethodcallindentation) version is nicer/easer to read. But happy to revert this if that view isn't shared.